validation/exclusions_test.go

```diff
@@ -56,8 +56,37 @@ var exclusions = map[string]map[uint64]bool{
 		11763072: true, // sepolia-dev-0/base-devnet-0     No Public RPC declared
 	},
 	"Standard_Contract_Versions": {
-		11155421: true, // sepolia-dev-0/oplabs-devnet-0
-		11763072: true, // sepolia-dev-0/base-devnet-0
+		10:        true, // mainnet/op           MCP (at time of writing)
```

Comment:
Do we mean to exclude this chain here? This is supposed to be a standard chain. Also for `sepolia/op`.